backend/src/lib/api-docs/constants.ts

```diff
@@ -614,3 +614,21 @@ export const INTEGRATION = {
     integrationId: "The ID of the integration object."
   }
 };
+
+export const AUDIT_LOG_STREAMS = {
+  CREATE: {
+    url: "The socket URL to push logs to.",
```

Comment:
I think there is some terminology mix up because we just send the logs via http, there is no socket or streams right?